wordpress-mobile/WordPress-Android
# Fixes: Title is not shown on My Site Dashboard with no sites & Tab height

Fixes #16154 & #16219

Since the changes were small and dependent on each other, this PR fixes the issue of toolbar height and margin and the 
No Title shown in Empty State(No Sites Present for an account). 


### My Site Dashboard when no sites are present  
<img src="https://user-images.githubusercontent.com/17463767/161026722-345eda50-03e3-4eb7-b40a-02081262df19.png" width="300" height="660">


Test 1 
- Launch app with an account having no sites
- Notice that **My Site** is shown in the toolbar 

Test 2
- Launch the app with an account having sites 
- Go to My Site
- Notice that the Site Info is shown above the quick links and collapses to Site title when scrolled below

Test 3
- Launch the app.
- Go to My Site -> Me -> App Settings -> Debug settings.
- Turn on the MySiteDashboardTabsFeatureConfig flag under the Features in development section.
- Restart the app
- Notice that the Site Info is shown above the tabs and collapses to Site title when scrolled below


## Regression Notes
1. Potential unintended areas of impact
My Site Dashboard not being shown properly when Sites are Present 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Testing 

3. What automated tests I added (or what prevented me from doing so)
None 

PR submission checklist:
